src/hooks/debounce.ts

```diff
@@ -1,12 +1,11 @@
 import { useEffect, useState } from "react";
 
-export default function useDebounce<T>(
-  value: T,
-  delay: number
-): {
+type UseDebounce = <T>(value: T, delay: number) => {
   debouncedValue: T;
   setDebouncedValue: React.Dispatch<React.SetStateAction<T>>;
-} {
+};
```

Comment:
Can this file be removed since we have imported a `use-debounce` package instead?

Reply:
sure, will be removed after refactoring all pages